src/core_plugins/kibana/public/discover/components/field_chooser/lib/detail_views/string.html

```diff
@@ -1,8 +1,8 @@
 <div class="discover-field-details">
   <div class="kuiVerticalRhythmSmall">
-    <h5 ng-show="!field.details.error">Quick Count <kbn-info info="Top 5 values based on documents in the table" placement="right"></kbn-info>
-      <span ng-if="!field.details.error" class="small discover-field-details-count">
-        (
+    <p class="kuiSubduedText" ng-show="!field.details.error">
```

Comment:
Let's add ` kuiSubText` class here to remove the bottom margin and preserve the font-size.